tests/cnf/core/network/internal/netparam/const.go

```diff
@@ -12,13 +12,13 @@ const (
 	// IPSubnet24 represents prefix 24 ipv4 subnet.
 	IPSubnet24 = "24"
 	// IPSubnet28 represents prefix 28 ipv4 subnet.
-	IPSubnet28 = "28"
+	IPSubnet28 = 28
 	// IPSubnet128 represents prefix 128 ipv6 subnet.
-	IPSubnet128 = "128"
+	IPSubnet128 = 128
 	// IPSubnet64 represents prefix 64 ipv6 subnet.
-	IPSubnet64 = "64"
+	IPSubnet64 = 64
 	// IPSubnet32 represents prefix 32 ipv4 subnet.
-	IPSubnet32 = "32"
+	IPSubnet32 = 32
 	// LogLevelDebug represents log level debug.
```

Comment:
What's the reason for this change?

Reply:
Because the actual variable type is a int and not string.  There was an issue with the nodeSelector test and I updated it.

Reply:
Right, but all the other tests that use strings and nodeSelector are working with only /32 subnets, correct?
So it’s probably better to just use an int in the nodeSelector tests instead of changing everything else.

```
	validatePrefix(frrPod0, netparam.IPV4Family, 32, removePrefixFromIPList(nodeAddrList), addressPool1)

	validatePrefix(frrPod1, netparam.IPV4Family, 32, removePrefixFromIPList(nodeAddrList), addressPool2)
```

Reply:
or convert string to int (num, err := strconv.Atoi(str))
or create a new var IPSubnet32Int = 32